### PR TITLE
security: add template for security advisory

### PR DESCRIPTION
**Pull request by snowp**
Risk Level: Low
Testing: n/a
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: n/a

### CODE REVIEW

File: security/email-templates.md [hunk 1]
```diff
@@ -206,3 +206,40 @@ Thanks,
...
+```
+Subject: Security advisory
+To: envoy-security-announce@googlegroups.com
+Cc: envoy-announce@googlegroups.com, envoy-security@googlegroups.com, envoy-maintainers@googlegroups.com
+
+Hello Envoy Community,
+
+The Envoy securitty team would like to announce a security advisory for a feature introduced in
+$ENVOYRELEASE. As this is a security advisory for a feature not considered production ready that may
+have been labeled as such, no fix is provided and the advisory is to not make use of this in a production
```
Comment (htuch) [inline security/email-templates.md:220]: `and the advice is to not make use of this feature in ...`

File: security/email-templates.md [hunk 2]
```diff
@@ -206,3 +206,40 @@ Thanks,
 
 $PERSON (on behalf of the Envoy security team and maintainers)
 ```
+
+## Security Advisory
```
Comment (htuch) [inline security/email-templates.md:210]: `## Security Advisory (for feature accidentally marked as production)`

**Review by htuch — COMMENTED** [review]
Thanks, feel free to minimize as much as you want.

**Comment by snowp** [comment]
@htuch ptal

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**